jupyter_collaboration/handlers.py

```diff
@@ -123,72 +48,43 @@ class YDocWebSocketHandler(WebSocketHandler, JupyterHandler):
        receiving a message.
     """
 
-    files: dict[str, FileLoader] = {}
-    websocket_server: JupyterWebsocketServer | None = None
     _message_queue: asyncio.Queue[Any]
 
-    def __init__(
-        self, app: ServerWebApplication, request: HTTPServerRequest, **kwargs: dict[str, Any]
+    def initialize(
+        self,
+        ywebsocket_server: JupyterWebsocketServer,
+        file_loaders: FileLoaderMapping,
+        ystore_class: Type[BaseYStore],
+        document_cleanup_delay: float | None = 60.0,
+        document_save_delay: float | None = 1.0,
```

Comment:
This looks like a code duplication from the traits, is it possible to use their values?

Reply:
Do you mean the default values for `document_cleanup_delay` and `document_save_delay`?

We are using the default values from the traits [here](https://github.com/jupyterlab/jupyter_collaboration/blob/298bf4466bdb6da42138847abae452351e419636/jupyter_collaboration/app.py#L92-L97).

I think it is OK to also have defaults in the header of the initialize method, just in case someone uses that class somewhere else.